modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -372,7 +372,7 @@ def empty(cls):
 
 
 @ray.remote(num_returns=2)
-def get_index_and_columns(df):
+def _get_index_and_columns(df):
```

Comment:
Implementation details should be private.
Same for `_apply_func`, `_apply_list_of_funcs`.